app/components/Headers/UnderlinedHeader/UnderlinedHeader.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React from 'react'
+
+import classNames from 'classnames'
+
+import styles from './UnderlinedHeader.scss'
+
+type Props = {
+  className?: string,
+  childClassName?: string,
+  text?: string,
+  children: React$Node
+}
+
+const UnderlinedHeader = ({
+  className,
+  childClassName,
+  text,
+  children
+}: Props) => (
+  <section className={classNames(styles.underlinedHeader, className)}>
+    <h1 className={classNames(styles.underlinedHeading, childClassName)}>
+      {text}
+    </h1>
+    {children}
+  </section>
+)
+
+export default UnderlinedHeader
```

Comment:
Theres actually a component for this that I created called `<HeaderBar/>` please use this instead and remove this component for consistency... Lets refactor `<HeaderBar/>` if necessary to meet your needs

Reply:
Cool no problem, will remove this component now